__tests__/app.test.js

```diff
@@ -59,20 +73,77 @@ describe("GET /api/articles/:article_id", () => {
             const article = response.body.article;
             //console.log(article);
             expect(typeof article).toBe("object");
+            expect(article).toHaveProperty("article_id");
+            expect(typeof article.article_id).toBe("number");
+            expect(article.article_id).toBe(1);
```

Comment:
Line 78 makes lines 76 and 77 a bit redundant, i.e. if `article.article_id` is 1, then you know it has that property and it's a number
Consider this throughout this test, which of these expects statements are necessary?